tests/unit/reports/test_carryforward.py

```diff
@@ -55,64 +53,8 @@ def sample_report():
     report.add_session(Session(id=0, flags=["simple"]))
     report.add_session(Session(id=1, flags=["complex"]))
     # manually fixing this because the defautl logic doesn't
-    report._files[
-        "file_1.go"
-    ].session_totals = SessionTotalsArray.build_from_encoded_data(
-        [
-            ReportTotals(
-                files=0,
-                lines=5,
-                hits=2,
-                misses=2,
-                partials=1,
-                coverage="40.00000",
-                branches=0,
-                methods=0,
-                messages=0,
-                sessions=0,
-                complexity=0,
-                complexity_total=0,
-                diff=0,
-            ),
-            ReportTotals(
-                files=0,
-                lines=5,
-                hits=2,
-                misses=3,
-                partials=0,
-                coverage="40.00000",
-                branches=0,
-                methods=0,
-                messages=0,
-                sessions=0,
-                complexity=0,
-                complexity_total=0,
-                diff=0,
-            ),
-        ]
-    )
-    report._files[
-        "file_2.py"
-    ].session_totals = SessionTotalsArray.build_from_encoded_data(
-        [
-            ReportTotals(
-                files=0,
-                lines=2,
-                hits=1,
-                misses=0,
-                partials=1,
-                coverage="50.00000",
-                branches=1,
-                methods=0,
-                messages=0,
-                sessions=0,
-                complexity=0,
-                complexity_total=0,
-                diff=0,
-            ),
-            None,
-        ]
-    )
+    report._files["file_1.go"]
```

Comment:
I think `_files` is just a straight `dict`, so accessing anything on it should be sideeffect free, so you can just remove the two lines completely.

Reply:
oh wow lol i really had tunnel vision deleting those references to not notice this